hijack/permissions.py

```diff
@@ -1,6 +1,6 @@
 def superusers_only(*, hijacker=None, hijacked=None):
```

Comment:
Looks like hijacker and hijacked can not be None

Reply:
No, that's a good point. I added another commit.